python3/libexec/nbd_client_manager.py

```diff
@@ -124,74 +134,101 @@ def _find_unused_nbd_device():
         nbd_device = "/dev/nbd{}".format(device_no)
         if not _is_nbd_device_connected(nbd_device=nbd_device):
             return nbd_device
-
+    # Actually `_is_nbd_device_connected` will raise an exception
+    # if no unused device
+    # Add this return for pylint check
+    return None
```

Comment:
Although it is impossible, but this return value would get the caller fail in error as the type of return doesn't match.

Reply:
It's impossible to have 1000 NBD devices and all of them are connected.
But you are right. Change this `return` to `raise exception`.